src/components/NavKnapp.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react';
+import { Flatknapp, Hovedknapp } from 'nav-frontend-knapper';
+import { useHistory } from 'react-router-dom';
+
+export enum knapptype {
+  Hoved = 'Hovedknapp',
+  Flat = 'Flatknapp',
+}
+
+interface Props {
+  tekstid: string;
+  type: knapptype;
+  nyPath: string;
+}
+
+const NavKnapp: FC<Props> = ({ tekstid, nyPath, type }) => {
+  const history = useHistory();
+  return (
+    <div className={'navknapp'}>
+      {type === knapptype.Hoved ? (
```

Comment:
Hvis poenget her er å ha en egen knapp for navigering så tenker jeg kanskje man heller burde bruke knappBase og sende inn type på den? Tror det skal være mulig. Isåfall trenger man kanskje ikke en egen komponent for å rendre 2 forskjellige knapper?

Reply:
Det har du rett i! Litt usikker på hva jeg tenkte når jeg lagde en annen NavKnapp for dette. 🤷‍Det ble overflødig, så bra at du så det 👍